src/main/java/Duke.java

```diff
@@ -1,10 +1,125 @@
+import java.util.Scanner;
 public class Duke {
+    private static final int LIST_SIZE = 100;
+    private static int taskNum = 0;
     public static void main(String[] args) {
+        showGreetings();
+        String command = "";
+        Scanner in = new Scanner(System.in);
+
+        Task[] storedTasks = new Task[LIST_SIZE];         // array of stored texts
+        while (!command.equals("bye")) {
+            printLineBreak();
+            command = in.nextLine();
+            String[] commandLine = command.split(" ", 2);
+
+            printLineBreak();
+
+            switch (commandLine[0]) {
+            case "todo":
+                addTodoTask(storedTasks, commandLine);
+                break;
+            case "deadline":
+                addDeadlineTask(storedTasks, commandLine);
+                break;
+            case "event":
+                addEventTask(storedTasks, commandLine);
+                break;
+            case "mark":
+                markTask(storedTasks, commandLine);
+                break;
+            case "unmark":
+                unmarkTask(storedTasks, commandLine);
+                break;
+            case "list":
+                listTasks(storedTasks);
```

Comment:
Good abstraction of functions, makes the code cleaner and easier to understand especially in the switch case statement!